File: 15552/15552.java
import java.io.*;

class Main{
	public static void main(String[] args) throws Exception{
		BufferedReader bfr = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bfw = new BufferedWriter( new OutputStreamWriter( System.out ) );
		int N =  Integer.parseInt(bfr.readLine());
		String [] splitNumber;
        Integer result;
		for(int i = 0; i<N;i++){
			bfw.flush();
			splitNumber = bfr.readLine().split(" ");
            result = Integer.parseInt(splitNumber[0])+Integer.parseInt(splitNumber[1]);
			bfw.write(result.toString());
            bfw.newLine();
		}
		bfw.close();
		bfr.close();
	}
}

File: 15552/15552.py
# # 내 코드
# import sys

# N = int(sys.stdin.readline())

# for i in range(0, N):
# 	A, B= sys.stdin.readline().split(' ')
# 	print(int(A)+int(B))

# 1등 코드 

from sys import stdin, stdout

input = stdin.readline
print = stdout.write

n = int(input())

for i in range(0, n):
	a, b = map(int, input().split())	
	print("%d\n"%(a+b))